content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -20,22 +20,21 @@ Use cases for raw SQL include:
 
 ## <inlinecode>queryRaw</inlinecode>
 
-`queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
+`$queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
 
 ```ts no-lines
 const result = await prisma.queryRaw('SELECT * FROM User;')
-```
```

Comment:
This removal breaks the page
https://5f296ef0f33d09000717f30d--prisma2.netlify.app/reference/tools-and-interfaces/prisma-client/raw-database-access#inlinecodequeryrawinlinecode
<img width="722" alt="Screen Shot 2020-08-04 at 16 26 28" src="https://user-images.githubusercontent.com/1328733/89305823-67d5cc00-d66f-11ea-981a-8b22307313ae.png">